regolith/helpers/u_milestonehelper.py

```diff
@@ -123,12 +124,20 @@ def db_updater(self):
         if rc.index > 1:
             doc = all_milestones[rc.index-2]
             identifier = doc['identifier']
+            if not rc.type and not doc.get('type'):
+                    raise RuntimeError(f"please rerun specifying --type with a value from {ALLOWED_TYPES}")
+            if rc.type:
+                try:
+                    doc.update({'type': ALLOWED_TYPES[rc.type]})
+                except:
```

Comment:
this should probably state the type of exception you are trying to catch, e.g.,`` except KeyError:`` ?  check and put it here.